03/app.js

```diff
@@ -2,4 +2,17 @@ document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     console.log('DOM');
+    const btn = document.querySelector('button');
+    btn.addEventListener('click', getIP);
+}
+
+async function getIP() {
+    const res = await fetch("https://api64.ipify.org?format=json");
+    const resInJson = await res.json();
```

Comment:
Jeśli używamy takiej konstrukcji to warto wykorzystać `try ... catch`, aby przechwycić ewentualny błąd :)